services/ui-src/src/components/pages/Profile/ProfilePage.tsx

```diff
@@ -0,0 +1,78 @@
+import { useNavigate } from "react-router-dom";
+import { Button, Heading, Link, Text } from "@chakra-ui/react";
+import { PageTemplate, Table } from "components";
+import { createEmailLink, useStore } from "utils";
+import verbiage from "verbiage/pages/profile";
+
+export const ProfilePage = () => {
+  const navigate = useNavigate();
+
+  const { email, given_name, family_name, userRole, state, userIsAdmin } =
+    useStore().user ?? {};
+
+  const { intro } = verbiage;
+
+  const tableContent = {
+    caption: "Profile Account Information",
+    bodyRows: [
+      ["Email", email!],
+      ["First Name", given_name!],
+      ["Last Name", family_name!],
+      ["Role", userRole!],
+      ["State", state! || "N/A"],
```

Comment:
I think this is the tiniest nit possible... This specific `!` isn't doing anything, since a falsey value will be replaced by the "N/A" anyway.
```suggestion
      ["State", state || "N/A"],
```

Reply:
or should i say it will be replaced "N/A-way"

Reply:
Clever Ben. N/A-way